int8 length
char[] pcm_buf
